sphinx/writers/manpage.py

```diff
@@ -139,6 +146,21 @@ def visit_desc_signature_line(self, node: Element) -> None:
     def depart_desc_signature_line(self, node: Element) -> None:
         self.body.append(' ')
 
+    def visit_desc_content(self, node: Element) -> None:
+        self.visit_definition(node)
+
+    def depart_desc_content(self, node: Element) -> None:
+        self.depart_definition(node)
+
```

Comment:
Is `*_desc_inline` needed here?

Reply:
That depends, I have added ``addnodes.desc_inline`` in ``SigElementFallbackTransform``, so technically it isn't needed. But if the goal is to minimize ``SigElementFallbackTransform``, then sure, I can add visit/depart for it as well.

Reply:
IMO, adding handlers makes our code explicit. So I prefer to add it here. The fallback transform is a bit magical and hard to understand its behavior on debugging.

Reply:
Indeed, I will add ``desc_inline`` handlers for the all the writers.